checkov/common/bridgecrew/integration_features/features/custom_policies_integration.py

```diff
@@ -86,10 +86,9 @@ def pre_scan(self) -> None:
                                 get_graph_checks_registry("kubernetes").checks.append(check)
                             elif f.lower() == "bicep":
                                 get_graph_checks_registry("bicep").checks.append(check)
-                    elif re.match(CFN_RESOURCE_TYPE_IDENTIFIER, check.resource_types[0]):
-                        get_graph_checks_registry("cloudformation").checks.append(check)
                     else:
-                        get_graph_checks_registry("terraform").checks.append(check)
+                        for registry in get_all_graph_checks_registries():
```

Comment:
note that this change basically changes the default behavior for custom policies.
I think we should control it with an env variable here, please add one (note to use `env_vars_config.py`)